lib/ecto/schema.ex

```diff
@@ -841,7 +843,8 @@ defmodule Ecto.Schema do
       `:delete_all` will only remove data from the join source, never the
       associated records. Notice `:on_delete` may also be set in migrations
       when creating a reference. If supported, relying on the database via
-      migrations is preferred
+      migrations is preferred. `:nilify_all` and `:delete_all` will not cascade to child
+      records unless set via database migrations.
```

Comment:
I think this is redundant in the docs for `many_to_many`. It already has this sentence: "`:delete_all` will only remove data from the join source, never the associated records."